cluster-autoscaler/debuggingsnapshot/debugging_snapshotter.go

```diff
@@ -230,6 +233,17 @@ func (d *DebuggingSnapshotterImpl) SetUnscheduledPodsCanBeScheduled(podList []*v
 	*d.State = DATA_COLLECTED
 }
 
+// SetTemplateNode is the setter for TemplateNode
+func (d *DebuggingSnapshotterImpl) SetTemplateNode(templates []*framework.NodeInfo) {
+	if !d.IsDataCollectionAllowed() {
+		return
+	}
+	klog.Infof("TemplateNode is being set for the debugging snapshot")
```

Comment:
Please use V() for any Info logs (here and elsewhere in this function). I'd consider a V(4) - on by default, we tend to use it as default log level for most logs.

Reply:
done